packages/frontend/package.json

```diff
@@ -15,6 +15,7 @@
     "vite": "^5.4.10"
   },
   "dependencies": {
+    "@word-games/backend": "workspace:*",
```

Comment:
client now imports backend in order to get router definitions - but we probably dont' want to import anything else directly form backend - gotta be careful to not import nodejs deps into the frontend.